src/runtime/manifest-parser.peg

```diff
@@ -126,6 +126,14 @@ Trigger "a trigger for a recipe"
   });
 }
 
+ParameterizedAnnotation "a parameterized annotation (e.g. @foo(bar))"
+  = simpleAnnotation:SimpleAnnotation '(' parameter:[a-z0-9_]i* ')' {
```

Comment:
It might be kind to developers to allow optional whitespace around the parameter. @SHeimlich

Reply:
I added the whitespace support.

Would you prefer a specific TtlAnnotation here, instead of a generic one? I was thinking this would be more reusable for other types of annotations we might want to add in the future (in accordance with https://docs.google.com/document/d/1OuPZYk6q-l0yxRiWzrE2H37vtRJQ0Qx6FhIkddhl_1c/edit#heading=h.ss4g0a9dnac1), but I'm happy to go the other way.

Reply:
Sorry thought I'd already responded to this. I don't think TtlAnnotation is needed, but separating the number from the units here (rather than in the JS would make more sense to me). Just a nit, this LGTM.